docs/apis-tools/grpc.md

```diff
@@ -1055,6 +1059,11 @@ Returned if:
 
 ### `DeleteResource` RPC
 
+:::warning
+This RPC is not tenant-aware. When multi-tenancy is enabled, resources can be
+deleted without considering the tenant ownership of the resource.
+:::
```

Comment:
❌ This isn't true is it? If multi-tenancy is enabled users can still only delete resources of the default tenant.

The way the processor works is it tries to find any process from de process state with the given key and tenant. If it finds any it deletes it.
If it doesn't find any it searches for a drg by tenant and key. If it finds one it deletes it.
If it doesn't find any we return a NOT_FOUND.

Obviously it'll take the tenant from the command. Since users can't set the tenant in the command I'd expect it to always get the value `<default>`.

Reply:
:+1: You have a point @remcowesterhoud. I'll adjust the docs.